src/Content/Backend/Solution/Monaco.Template.Backend.Api/Endpoints/Products.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Asp.Versioning.Builder;
+using MediatR;
+using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.AspNetCore.Mvc;
+#if (!disableAuth)
+using Monaco.Template.Backend.Api.Auth;
+#endif
+using Monaco.Template.Backend.Api.DTOs;
+using Monaco.Template.Backend.Api.DTOs.Extensions;
+using Monaco.Template.Backend.Application.DTOs;
+using Monaco.Template.Backend.Application.Features.Product;
+using Monaco.Template.Backend.Common.Api.Application;
+using Monaco.Template.Backend.Common.Api.MinimalApi;
+using Monaco.Template.Backend.Common.Domain.Model;
+
+namespace Monaco.Template.Backend.Api.Endpoints;
+
+public static class Products
+{
+	public static IEndpointRouteBuilder AddProducts(this IEndpointRouteBuilder builder, ApiVersionSet versionSet)
+	{
+		var products = builder.CreateApiGroupBuilder(versionSet, "Products");
+
+		products.MapGet("",
```

Comment:
Perhaps it is worth adding another overload of the `Map{Verb}` extension methods without the `string pattern` parameter that then calls another overload passing `string.Empty` or `""` for it's `pattern` parameter? What do you think?

Reply:
Would you mind to suggest some snippet of code about what you mean?
That would be easier :)

Reply:
Hmmm... I think I would prefer to leave it explicit for the dev to set the empty string there...
When someone takes a quick look at the same position of each endpoint, everyone usually attempts to find the string written there in order to get an idea of what is the actual route that endpoint is pointing too... I think taking it out might make it less obvious and is something where being a little extra verbose could be beneficial for the sake of quick reading the endpoints.

![image](https://github.com/onebeyond/monaco/assets/3091332/754c4f15-5c5b-4d34-87b5-09ffc6ab5787)

There might be more endpoints and having to switch between overloads with and without pattern parameter there, might be like climbing a stair with the steps of different dimensions 😅